src/users/users.controller.spec.ts

```diff
@@ -1,14 +1,16 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { UsersController } from './users.controller';
 import { UsersService } from './users.service';
+import { DbConnectionService } from '../db/db.service';
+import { ConfigService } from '@nestjs/config';
 
 describe('UsersController', () => {
   let controller: UsersController;
 
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
       controllers: [UsersController],
-      providers: [UsersService],
+      providers: [UsersService, DbConnectionService, ConfigService],
```

Comment:
Is this the best way of doing this? Or are we supposed to mock the `DbConnection` and `Config` services?